react/NestedSelect/Row.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Media, Bd, Img, Icon } from 'cozy-ui/transpiled/react'
+import styles from './Row.styl'
+import cx from 'classnames'
+
+export const RowBody = ({ children }) => (
+  <Bd className="u-ellipsis">{children}</Bd>
+)
+
+const Row = ({
+  isSelected,
+  icon,
+  label,
+  children,
+  hasArrow,
+  onClick,
+  className
+}) => (
+  <Media
+    className={cx(
+      styles.Row,
+      'u-row-m',
```

Comment:
Where is this defined?

Reply:
You mean u-row-m ? https://github.com/cozy/cozy-ui/blob/master/stylus/utilities/stack.styl